applications/GeoMechanicsApplication/python_scripts/geomechanics_solvers_wrapper.py

```diff
@@ -30,6 +35,10 @@ def CreateSolver(model, custom_settings):
         raise Exception(err_msg)
 
     module_full_name = 'KratosMultiphysics.GeoMechanicsApplication.' + solver_module_name
-    solver = import_module(module_full_name).CreateSolver(model, custom_settings["solver_settings"])
-
+    solver = import_module(module_full_name).CreateSolver(model, params)
     return solver
+
+
+def CreateSolver(model, custom_settings):
+    parallelism =  custom_settings["problem_data"]["parallel_type"].GetString()
```

Comment:
Very minor formatting suggestion
```suggestion
    parallelism = custom_settings["problem_data"]["parallel_type"].GetString()
```